pkg/gator/test/test.go

```diff
@@ -185,3 +187,13 @@ func makeRegoDriver(tOpts Opts) (*rego.Driver, error) {
 
 	return rego.New(args...)
 }
+
+func makeK8sCELDriver(tOpts Opts) (*k8scel.Driver, error) {
+	var args []k8scel.Arg
+
+	if tOpts.GatherStats {
+		args = append(args, k8scel.GatherStats())
+	}
+
+	return k8scel.New(args...)
+}
```

Comment:
nit new line

Reply:
might be good to also add this flag to https://open-policy-agent.github.io/gatekeeper/website/docs/gator in case users want to disable it from the default.

Reply:
I will update docs once we cut the release. Otherwise these changes will be part of 3.19.